### PR TITLE
Update stage 3

### PR DESCRIPTION
**Pull request by vic-ma**

### CODE REVIEW

File: course-definition.yml [hunk 1]
```diff
@@ -168,60 +168,61 @@ stages:
...
+      ### Request message
 
-      In the previous stage, we saw that the request starts with a 4 byte length field, followed by the request header, and then the request body.
+      A request message has three parts:
+      1. Message size
+      2. Header
+      3. Body
 
-      The format of the request header is as follows:
+      To get the correlation ID, you need to find its offset. You already know that the message size is 4 bytes long. And here's what the request header looks like (in this challenge, we're using [request header v2](https://kafka.apache.org/protocol.html#protocol_messages)):
```
Comment (vic-ma) [inline course-definition.yml:180]: Should this also be changed to "In this stage," as per [this comment](https://linear.app/codecrafters/document/kafka-stage-2-ab182594c97e#comment-01c552eb)?

**Comment by vic-ma** [comment]
@rohitpaulk Ready for review

File: course-definition.yml (hunk 1 above)
Comment (rohitpaulk) [reply course-definition.yml:180]: Yep!

Comment (vic-ma) [reply course-definition.yml]: @rohitpaulk done!

**Pull request merged**